src/main/java/frc/robot/utils/math/EqualsUtil.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
Same here, we should give a link to the original file.  I've added comments to Anshul's use of this file, and I've rolled it into the BaseCode, so I won't repeat those comments here.